### PR TITLE
Documentation for 0.0.5

### PR DESCRIPTION
**Pull request by christiam**

### CODE REVIEW

File: docs/source/configuration.rst [hunk 1]
```diff
@@ -263,6 +316,22 @@ BLAST configuration options
     [blast]
     batch-len = 100000
 
+.. _elb_blast_db_margin:
+
+``BLAST database memory margin`` 
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+    Source from where to get the BLAST databases.
```
Comment (boratyng) [inline docs/source/configuration.rst]: That looks like a copy/paste problem.

Comment (christiam) [reply docs/source/configuration.rst]: Indeed, thanks for catching that. I committed a fix for it.

**Review by tom6931 — COMMENTED** [review]
I disagree with the defaults for BLAST database memory margin.  If I understand this correctly, you'll need a machine with 245G (223G*1.1) to run a search against nr.  I've been running successfully with the 120G on n1-standard-32.  What you basically need is enough memory to keep the psq and pin files in memory (100G), so memory margin should be 0.44.  Is this a new option?  It needs to be tested.  Right now, it should be set lower or disabled.  Let me know if I misunderstand what this is.

Under Requirements: right now, only python3.6 works for me.  We should say this unless we can support other versions.  Also, there is a link to BLAST+ like it's required, but it's not.  At least, I think it's not.  This should be removed.

I'm no longer convinced that 100,000 is a good value for batch length for BLASTP.  I would put that lower (40,000) unless there are tests that show it's OK

Why do we provide a BLAST database source?  We provide the same databases at AWS, FTP, and GCP so users can get the same databases no matter where they are.  Exposing this is only going to slow people down or cause them problem (best case scenario) or end up hitting some account with big egress charges (worst case).  We should hide this option.

File: docs/source/configuration.rst [hunk 2]
```diff
@@ -263,6 +316,22 @@ BLAST configuration options
     [blast]
     batch-len = 100000
 
+.. _elb_blast_db_margin:
+
+``BLAST database memory margin`` 
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+    This value specifies how much larger should the `Memory request for BLAST search`_  be made relative to the size of the `BLAST database`_ by default.
```
Comment (boratyng) [inline docs/source/configuration.rst:331]: I do not think this is true, see `src/elb/commands/submit.py:140`. How about this: This value specifies minimum memory for cluster worker nodes relative to database size required for Elastic-BLAST to run.

**Review by boratyng — COMMENTED** [review]
> I'm no longer convinced that 100,000 is a good value for batch length for BLASTP. I would put that lower (40,000) unless there are tests that show it's OK

The current default batch length for blastp in Elastic-BLAST is 10,000. Most dark matter searches run under 40 minutes with this batch length. I would not go larger right now without a few experiments.

**Comment by boratyng** [comment]
> I disagree with the defaults for BLAST database memory margin. If I understand this correctly, you'll need a machine with 245G (223G*1.1) to run a search against nr. I've been running successfully with the 120G on n1-standard-32. What you basically need is enough memory to keep the psq and pin files in memory (100G), so memory margin should be 0.44. Is this a new option? It needs to be tested. Right now, it should be set lower or disabled. Let me know if I misunderstand what this is.

I think we should disable this functionality until we have a better idea of how to estimate memory requirement.

**Comment by christiam** [comment]
> > I'm no longer convinced that 100,000 is a good value for batch length for BLASTP. I would put that lower (40,000) unless there are tests that show it's OK
> 
> The current default batch length for blastp in Elastic-BLAST is 10,000. Most dark matter searches run under 40 minutes with this batch length. I would not go larger right now without a few experiments.

The batch length for blastp has been corrected to 10,000 in 75bb0e8.

**Comment by christiam** [comment]
> Also, there is a link to BLAST+ like it's required, but it's not. At least, I think it's not. This should be removed.

Per the latest code changes, this is no longer needed and I updated the requirements accordingly.

**Comment by christiam** [comment]
> Why do we provide a BLAST database source? We provide the same databases at AWS, FTP, and GCP so users can get the same databases no matter where they are. Exposing this is only going to slow people down or cause them problem (best case scenario) or end up hitting some account with big egress charges (worst case). We should hide this option.

Done in d845c35

**Comment by christiam** [comment]
> I think we should disable this functionality until we have a better idea of how to estimate memory requirement.

FTR this is done.

**Review by tom6931 — APPROVED** [review]

**Pull request merged**